docs/index.html

```diff
@@ -51,17 +51,6 @@ <h3 class="sidebar-header">COMPONENTS</h3>
             <li class="docs-navbar-item" target="alert">
               <a href="#" target="alert">Alerts</a>
```

Comment:
target allows only below values and also we don't need it here in the nav

![image](https://user-images.githubusercontent.com/67600170/152677047-09d7376e-cdd2-4ad9-b065-2179bfa95483.png)